view/frontend/web/css/source/_module.less

```diff
@@ -67,17 +67,17 @@
                     margin-left: 5px;
                 }
 
+                .rvvup-pay-by-bank-img,
```

Comment:
Seems like we had this wrong in the first place (should be pay_by_bank,)

Also looks PHP cs doesn't like _ in class names 😂

Reply:
Looks like we have .rvvup-payment-logo on all our logos, but it was getting overridden by the `data-bind` so was being removed. if we add it in the data bind along with our `rvvup-*-img` then we can remove these individual ones and just have a style for `rvvup-payment-logo`.